packages/dashboard/src/customization/widgets/lineSymbol/component.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { CSSProperties } from 'react';
+import { LineWidget } from '../types';
+import { SVG_STROKE_DASHED, SVG_STROKE_DOTTED, SVG_STROKE_SOLID } from '../constants';
+import { useWidgetActions } from '~/customization/hooks/useWidgetActions';
+import { DashboardWidget } from '~/types';
+import { LineAnchor } from './lineAnchor/component';
+import { useIsSelected } from '~/customization/hooks/useIsSelected';
+
+const LineWidgetComponent: React.FC<LineWidget> = (widget) => {
+  const { update } = useWidgetActions<DashboardWidget>();
+  const isSelected = useIsSelected(widget);
+
+  const fitFull = {
```

Comment:
nit: seems like a constant which need not be created every time. i.e could be outside the component

Reply:
also, this could be just a class? instead of a style variable

Reply:
done - moved it to a css file and used it as a class